git-config/src/values.rs

```diff
@@ -4,6 +4,7 @@ use std::{borrow::Cow, convert::TryFrom, fmt::Display, str::FromStr};
 
 #[cfg(feature = "serde")]
 use serde::{Serialize, Serializer};
+use std::path::{Path as StdPath, PathBuf};
```

Comment:
I would go with fully qualified paths if there is a conflict, like `std::path::Path`.